### DISCUSSION TITLE
Quarto and JS documentation

### DISCUSSION THREAD

**Discussion by BatoolMM** [quarto-dev/quarto-cli#5061] [Q&A]
Is it possible to use Quarto to document our JavaScript API? We are developing open educational resources using MongoDB, Node.js, Express.js, and React, as described in [our repository](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs). Currently, we are using JSDoc to document the API, but we are wondering if Quarto can be used for this purpose?

**Comment by mcanouil** [comment]
You can use Quarto for whatever you want.
For example, there is `quartodoc` to generate Python documentation with Quarto.
I am not aware of something already created for JavaScript.
